src/vec8bit.c

```diff
@@ -1353,9 +1363,14 @@ Obj FuncSUM_VEC8BIT_VEC8BIT( Obj self, Obj vl, Obj vr)
 
         /* if the exponent is bigger than 31, overflow changes the value to 0 */
         if (newd > 8 || newq > 256 ||
-            (ql != newq && True == CALL_1ARGS(IsLockedRepresentationVector, vl)) ||
-            (qr != newq && True == CALL_1ARGS(IsLockedRepresentationVector, vr))) {
+            (ql != newq && !MayRewrite(vl)) ||
+            (qr != newq && !MayRewrite(vr))) {
             sum = SumListList(vl, vr);
+	    if (newq <= 256)
+	      if (REGION(sum))
+		ConvVec8Bit(sum,newq);
+	      else
+		sum = NewVec8Bit(sum,newq);
             return sum;
```

Comment:
Perhaps we have to build GAP with
```
make COPTS="-Werror"
```
to trigger Travis CI build failure in case of compiler warnings, and not to wait till we will merge PR and then detect them with Jenkins...